crdt/woot.go

```diff
@@ -38,11 +41,44 @@ var (
 	ErrBoundsNotPresent    = errors.New("subsequence bound(s) not present")
 )
 
-// New returns a initialized document.
+// New returns an initialized document.
 func New() Document {
 	return Document{Characters: []Character{CharacterStart, CharacterEnd}}
 }
 
+// Load reads a text file from disk and converts it into a CRDT document.
+func Load(fileName string) (Document, error) {
+	doc := New()
+	b, err := os.ReadFile(fileName)
+	if err != nil {
+		return doc, err
+	}
+	lines := strings.Split(string(b), "\n")
+	pos := 1
+	for i := 0; i < len(lines); i++ {
+		for j := 0; j < len(lines[i]); j++ {
+			_, err := doc.Insert(pos, string(lines[i][j]))
+			if err != nil {
+				return doc, err
+			}
+			pos++
+		}
+		if i < len(lines)-1 {
+			_, err := doc.Insert(pos, "\n")
+			if err != nil {
+				return doc, err
+			}
+			pos++
+		}
```

Comment:
Nice addition! Would love to see this in action; you could add this a separate test as a part of the unit tests.